etc/config/c++.amazon.properties

```diff
@@ -826,6 +826,12 @@ libs.ctre.versions.trunk.version=trunk
 libs.ctre.versions.trunk.path=/opt/compiler-explorer/libs/ctre/master/include
 libs.ctre.versions.v2.version=v2
 libs.ctre.versions.v2.path=/opt/compiler-explorer/libs/ctre/v2/include
+libs.spy.name=SPY
+libs.spy.description=C++17 constexpr settings detectors
+libs.spy.versions=v003
+libs.spy.url=https://github.com/jfalcou/spy
+libs.spy.versions.v003.version=v0.0.3
+libs.spy.versions.v003.path=/opt/compiler-explorer/libs/spy/include
```

Comment:
the path is `/opt/compiler-explorer/libs/spy/0.0.3/include`

(and for trunk substitute `trunk` for `0.0.3`)